apps/mark/frontend/src/app_root.tsx

```diff
@@ -534,15 +530,10 @@ export function AppRoot({
     resetBallotOnLogout();
   }, [auth, resetBallot, initializedFromStorage]);
 
-  const markVoterCardVoided: MarkVoterCardFunction = useCallback(() => {
+  const endVoterSession = useCallback(() => {
     assert(isCardlessVoterAuth(auth));
     auth.logOut();
-    return Promise.resolve(true);
-  }, [auth]);
-
-  const markVoterCardPrinted: MarkVoterCardFunction = useCallback(() => {
-    assert(isCardlessVoterAuth(auth));
-    return Promise.resolve(true);
+    return Promise.resolve();
```

Comment:
Kept this async because the equivalent of `auth.logOut` once I move VxMark auth to the backend will be async